### PR TITLE
chore(shorebird_cli): use upstream googleapis_auth package

### LINKED ISSUE #1944: fix: Because shorebird_cli depends on googleapis_auth from git which doesn't exist
**Description**

A clear and concise description of what the bug is when install shorebird in windows 10 by 

```
Set-ExecutionPolicy RemoteSigned -scope CurrentUser # Needed to execute remote scripts
iwr -UseBasicParsing 'https://raw.githubusercontent.com/shorebirdtech/install/main/install.ps1'|iex
```

Got error 

Because shorebird_cli depends on googleapis_auth from git which doesn't exist
  (Could not find git ref '647f96d5ffdae63e41aeea091c34acfffff533e7' (fatal: bad
  object 647f96d5ffdae63e41aeea091c34acfffff533e7)),

**Steps To Reproduce**

**Expected Behavior**

**Screenshots**

image

**Additional Context**

### PR DESCRIPTION
**Pull request by bryanoltman**
## Description

Fixes https://github.com/shorebirdtech/shorebird/issues/1944

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [x] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [ ] 🗑️ Chore

### CODE REVIEW

**Comment by eseidel** [comment]
Does this bump our minimum Dart version?

**Review by felangel — APPROVED** [review]

File: packages/shorebird_cli/pubspec.yaml [hunk 1]
```diff
@@ -26,11 +26,7 @@ dependencies:
   cli_util: ^0.4.1
   collection: ^1.18.0
   crypto: ^3.0.3
-  googleapis_auth:
-    git:
-      url: https://github.com/shorebirdtech/googleapis.dart.git
-      path: googleapis_auth
-      ref: 647f96d5ffdae63e41aeea091c34acfffff533e7
+  googleapis_auth: ^1.6.0
```
Comment (felangel) [inline packages/shorebird_cli/pubspec.yaml:29]: Don't we also need to bump the min dart sdk version?

Comment (felangel) [reply packages/shorebird_cli/pubspec.yaml:29]: nevermind just checked and it looks like googleapis_auth min is 3.0.0 so we're good 👍

Comment (bryanoltman) [reply packages/shorebird_cli/pubspec.yaml:29]: I think there may a dependency of that project that requires a higher version. Checking with an older Dart version now.

Comment (bryanoltman) [reply packages/shorebird_cli/pubspec.yaml:29]: It looks like several of our dependencies require newer versions of Dart than our pubspec currently says it requires. `http 1.2.1` requires Dart 3.3.0, for example.

**Comment by eseidel** [comment]
We already pull our own Dart.  I'm not sure what call we're making that's not using it:
https://github.com/shorebirdtech/shorebird/blob/main/bin/shorebird
https://github.com/shorebirdtech/shorebird/blob/main/third_party/flutter/bin/internal/shared.sh

**Comment by eseidel** [comment]
Yeah, we have our own Dart, we should just fix cases where we're using the wrong one.

**Pull request merged**